skriptpy/__init__.py

```
@@ -0,0 +1,5 @@
+# skriptpy/__init__.py
+__all__ = ["event", "command", "send", "broadcast", "teleport", "set_var", "get_var", "player", "If", "Else", "loop_times", "loop_players"]
```

Ruff (0.8.2)

`event` may be undefined, or defined from star imports

(F405)